workflows/rules/variantcalling/tnscope.smk

```diff
@@ -43,19 +50,24 @@ else:
         singularity:
             pipeconfig["singularities"]["sentieon"]["sing"]
         output:
-            tnscope = "{workingdir}/{stype}/tnscope/{sname}_TNscope_tn.vcf",
-            tnscope_bam = "{workingdir}/{stype}/tnscope/{sname}_REALIGNED_realignedTNscope.bam"
+            tnscope_vcf = temp("{stype}/tnscope/{sname}_TNscope_tn.vcf"),
+            tnscope_idx = temp("{stype}/tnscope/{sname}_TNscope_tn.vcf.idx"),
```

Comment:
were these extra files just assumed to be there earlier?

Reply:
Yes. Similar to the `.bai`, the `.idx` files are automatically generated, but need to be explicitly used as inputs.